src/main/java/nextstep/qna/domain/Answer.java

```diff
@@ -72,6 +68,16 @@ public void toQuestion(Question question) {
         this.question = question;
     }
 
+    public DeleteHistory delete(NsUser loginUser) throws CannotDeleteException {
+        if (!isOwner(loginUser)) {
+            throw new CannotDeleteException("다른 사람이 쓴 답변이 있어 삭제할 수 없습니다.");
+        }
+
+        this.deleted = true;
+
+        return new DeleteHistory(ContentType.ANSWER, this.id, this.writer, LocalDateTime.now());
```

Comment:
```suggestion
        return DeleteHistory.createAnswer(this.id, this.writer, LocalDateTime.now());
```

삭제이력은 답변, 질문 모두 있으니
위와 같이 정적생성자를 사용해도 될것 같아요!

Reply:
네, 객체 생성 로직을 드러내기 보다 캡슐화하는게 나아보이네요. 감사합니다!!